source/logs/location.txt

```diff
@@ -0,0 +1,80 @@
+.. _mongosh-log-location:
+
+=========================
+Specify Log File Location
+=========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can specify where MongoDB Shell writes log files. By default,
+MongoDB Shell saves the log for each session to your user's
+``.mongodb/mongosh`` directory, which depends on your operating system:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Operating System
+     - Default Log Location
+   * - macOS and Linux
+     - ``~/.mongodb/mongosh/<LogID>_log``
+   * - Windows
+     - ``%UserProfile%/AppData/Local/mongodb/mongosh/<LogID>_log``
```

Comment:
Maybe it's worth mentioning that files are written using a `mongosh_` file prefix if and only if the user has set an explicit directory using `logLocation`?

Reply:
Good idea! I wasn't aware of this behavior.